install/share/bind.named.conf.template

```diff
@@ -4,9 +4,9 @@ options {
 
 	// Put files that named is allowed to write in the data/ directory:
 	directory "$NAMED_VAR_DIR"; // the default
-	dump-file		"data/cache_dump.db";
-	statistics-file		"data/named_stats.txt";
-	memstatistics-file	"data/named_mem_stats.txt";
+	dump-file		"$DATA_DIRcache_dump.db";
```

Comment:
It's really hard to read and probably does not work as intended. https://docs.python.org/3/library/string.html#template-strings supports ```${id}``` syntax, so ```${DATA_DIR}cache_dump.db```